features/net/FEATURE_IPV4/lwip-interface/.mbedignore

```diff
@@ -0,0 +1,6 @@
+lwip/doc/*
+lwip/test/*
+lwip/src/apps/*
+lwip/src/netif/ppp/*
+lwip/src/inlude/lwip/apps/*
+lwip/src/inlude/posix/*
```

Comment:
Would mean writing special scripts and tools to cope with the variation, and more error-prone tracking of the upstream. This is attempting to work with the source control system rather than against it.  (As much as we can given the limitations of mbed-os).

This way means it's just a single "git subtree add".